### ISSUE TITLE
o-overlay: invalid heading role used in demo - ID: DAC_Name_Role_Value_01

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami#556]
# Issue
This is from page 183 of the [audit report](https://drive.google.com/file/d/17wq6b0CIX41RA4muBFDzUDxE0-qzHBIH/view)

Issue can be found on: <https://www.ft.com/__origami/service/build/v3/demo?component=o-overlay@4.0.0&demo=modal&system_code=origami-repo-data&brand=master>

The ‘We’re not having trouble updating’ text has been assigned a role of ‘heading’, but the role has is invalid as there is no aria-level to describe the level of the heading. This issue may affect screen reader users who may be unable to identify the text as a heading in their journey.

Additional instances of this issue exist on other pages throughout the website; wherever this issue occurs, they too will need to be resolved.
URL: <https://www.ft.com/__origami/service/build/v3/demo?component=o- overlay%404.0.0&demo=modal-fullscreen&system_code=origami-repo-data&brand=master>
Journey 49.

# Proposed Solution:
Ensure that if the role of heading is used that the aria-level attribute is assigned to allow screen reader users to identify the role and the heading level correctly. Alternatively, use the native HTML heading tag to display the heading.

**Comment by JakeChampion** [comment]
This was fixed in https://github.com/Financial-Times/origami/commit/a41eb3aaaf6c43a3e2839d00263cdaaa08ee735f
